backend/entityservice/tests/test_project_uploads.py

```diff
@@ -235,3 +235,62 @@ def test_project_single_party_empty_data_upload(
     )
     assert r.status_code == 400
 
+
+def test_project_upload_wrong_authentication(requests, valid_project_params):
```

Comment:
you can be more specific here. You are testing that you cannot upload twice.

Reply:
:+1: